htdocs/compta/facture/list.php

```diff
@@ -589,7 +589,7 @@
 	$massactionbutton=$form->selectMassAction('', $massaction ? array() : array('presend'=>$langs->trans("SendByMail")));
     
     $i = 0;
-    print '<form method="POST" name="searchFormList" action="'.$_SERVER["PHP_SELF"].'">'."\n";
+    print '<form method="GET" name="searchFormList" action="'.$_SERVER["PHP_SELF"].'">'."\n";
```

Comment:
Hi, cause of print button. It use the QUERY_STRING to compose it, i do another PR on develop to manage POST in